src/Microphysics2M.jl

```diff
@@ -545,12 +544,37 @@ function effective_radius(
     Br =
         (xr == 0) ? 0 :
         ((SF.gamma((νr + 1) / μr) * xr / SF.gamma((νr + 2) / μr))^(-μr))
+    Ac = (N_liq * Bc^(νc + 1)) / SF.gamma(νc + 1)
+    Ar = (μr * N_rai * Br^((νr + 1) / μr)) / SF.gamma((νr + 1) / μr)
+
+    cloud_3moment = (Bc == 0) ? 0 : (6 * Ac * C^3 / (Bc * C)^4)
+    cloud_2moment =
+        (Bc == 0) ? 0 : (Ac * SF.gamma(11 / 3) / (Bc^(11 / 3) * C^(2 / 3)))
```

Comment:
The type of `(1/3)` etc will be `Float64`, so we should wrap all of them in `FT()`. 

It's not super important here, because both `Z` and `r_eff` will be used as diagnostics. But just in case